mail_edit/static/src/xml/mail_edit.xml

```diff
@@ -6,6 +6,9 @@
             <span class="oe_edit">
                 <a title="Edit Mail" class="oe_e">&#038;</a>
             </span>
+            <span class="oe_delete">
```

Comment:
how about ``t-if="widget.is_author``? Or even ``t-att-disabled="widget.is_author None or 'disabled'"`` (from the top off my head, untested). Then there's no need to check for author above